### PR TITLE
Remove gcs-fetcher from tektoncd-pipeline

### PR DESCRIPTION
**Pull request by pradeepitm12**
remove gcs-fetcher from tektoncd-pipeline
also remove release-netxt__4.5

### CODE REVIEW

**Comment by piyush-garg** [comment]
/hold

Why you want to change in v0.20.1 and 0.21.0 ?

**Comment by pradeepitm12** [comment]
> /hold
> 
> Why you want to change in v0.20.1 and 0.21.0 ?
This is my bad, will update this pr.

File: ci-operator/config/openshift/tektoncd-pipeline/openshift-tektoncd-pipeline-release-next__4.7.yaml [hunk 1]
```diff
@@ -85,14 +85,6 @@ images:
...
-- dockerfile_path: openshift/ci-operator/tekton-images/gcs-fetcher/Dockerfile
-  from: ubi-minimal
-  inputs:
-    bin:
-      paths:
-      - destination_dir: .
-        source_path: /go/bin/.
-  to: tektoncd-pipeline-gcs-fetcher
 promotion:
   name: tektoncd-next-4.7
```
[the remark this answers is not in the document]
Comment (pradeepitm12) [reply ci-operator/config/openshift/tektoncd-pipeline/openshift-tektoncd-pipeline-release-next__4.7.yaml]: Sure.
Done

**Comment by savitaashture** [comment]
@pradeepitm12 i see build failure error related to gcs-fetcher

**Pull request merged**